fxa-auth-db-server/lib/bufferize.js

```diff
@@ -15,21 +13,24 @@ function unbuffer(object) {
   return object
 }
 
-function bufferize(object, ignore) {
+function bufferize(object, onlyTheseKeys) {
   var keys = Object.keys(object)
+  if (onlyTheseKeys) {
+    keys = keys.filter(function(key) {
```

Comment:
...similarly, this chap is *aching* to be an arrow function if we want to move in that direction.